articles/firebase-email-and-password-authentication-in-android-using-kotlin/index.md

```diff
@@ -0,0 +1,728 @@
+# Introduction
+
+[Firebase authentication](https://firebase.google.com/docs/auth) is a technology used to set up service-access permissions. It does so by creating and managing valid user accounts using firebase APIs and Firebase console provided by Google. In this tutorial, you'll learn how to implement firebase authentication in an android application.
+
+## Prerequisites
+
+To follow along with this tutorial you're required to have:
+
+- [Android Studio IDE](https://developer.android.com/studio) installed in your machine
+- Knowledge on [constraint layout](https://developer.android.com/training/constraint-layout) in XML
+- An active [Google account](https://accounts.google.com/signup/v2/webcreateaccount?flowName=GlifWebSignIn&flowEntry=SignUp)
+
+### Step 1 -- Creating An Android project
+
+First, we need to create an Android project that we will use in the entire tutorial. To do this, launch the android studio and you'll see a welcome screen similar to the one below.
+
+![image](android_studio_welcome_screen.png "Android studio home screen")
```

Comment:
Use this format for adding image links (/engineering-education/<article folder name>/<image name>)

Reply:
After doing this, images appear broken in the preview. Is it still okay ?

Reply:
Yes, it is okay.